src/plugins/expressions/common/executor/executor.ts

```diff
@@ -257,6 +257,21 @@ export class Executor<Context extends Record<string, unknown> = Record<string, u
     return telemetryData;
   }
 
+  migrate(ast: SerializableState, version: string) {
+    return this.walkAst(cloneDeep(ast) as ExpressionAstExpression, (fn, link) => {
```

Comment:
we know the input is ExpressionAstExpression as we don't allow migrating that structure but just the actual functions within